libfwupdplugin/fu-volume.c

```diff
@@ -81,7 +81,7 @@ fu_volume_get_id (FuVolume *self)
 gchar *
 fu_volume_get_mount_point (FuVolume *self)
 {
-	const gchar **mountpoints = NULL;
+	g_autofree const gchar **mountpoints = NULL;
```

Comment:
With this change, does it make sense to perhaps `g_steal_pointer()` on the function return?

Reply:
No, as it's only returning `container` and we're returning one of the array _elements_ that is returned const.